<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBox</class>
 <widget class="QGroupBox" name="GroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Image Source</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="serverLabel">
       <property name="toolTip">
        <string>detector tangohost name</string>
       </property>
       <property name="text">
        <string>Server:</string>
       </property>
       <property name="buddy">
        <cstring>serverComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="serverComboBox">
       <property name="toolTip">
        <string>detector tangohost name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="attrLabel">
       <property name="toolTip">
        <string>tango device name with its attribute, e.g. sys/tg_test/1/double_image_ro</string>
       </property>
       <property name="text">
        <string>Attribute:</string>
       </property>
       <property name="buddy">
        <cstring>attrLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="fileLineEdit">
       <property name="toolTip">
        <string>tango device attribute with its filename, e.g. p00/pilatus/1/LastImageTake</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="dirLineEdit">
       <property name="toolTip">
        <string>tango device attribute with its directory (optional), e.g. p00/pilatus/1/LastImagePath</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="pickleLabel">
       <property name="toolTip">
        <string>zmq server, port and topic, hwm (optional): server:port[/topic][/hwm]
e.g. haso228:9999/10001 or :5553</string>
       </property>
       <property name="text">
        <string>ZMQ Server:</string>
       </property>
       <property name="buddy">
        <cstring>pickleLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="cStatusLineEdit">
       <property name="styleSheet">
        <string notr="true">color: blue; background-color: yellow;</string>
       </property>
       <property name="text">
        <string>Not connected</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>start/stop reading images</string>
       </property>
       <property name="text">
        <string>&amp;Start</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="pickleTopicComboBox">
       <property name="toolTip">
        <string>ZMQ stream topic</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="httpLineEdit">
       <property name="toolTip">
        <string>monitor url address or hostname/api_version, e.g. http://haso228eiger/monitor/api/1.5.0/images/monitor or haso228eiger/1.5.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="currenthostLabel">
       <property name="toolTip">
        <string>current host name and the hidra port</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="attrLineEdit">
       <property name="toolTip">
        <string>tango device name with its attribute, e.g. sys/tg_test/1/double_image_ro</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="hostLabel">
       <property name="toolTip">
        <string>current host name and the hidra port</string>
       </property>
       <property name="text">
        <string>Client:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sourceTypeLabel">
       <property name="toolTip">
        <string>image source type, e.g. Hidra, Tango, ZMQ Stream HTTP response, Test</string>
       </property>
       <property name="text">
        <string>Source:</string>
       </property>
       <property name="buddy">
        <cstring>sourceTypeComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="fileLabel">
       <property name="toolTip">
        <string>tango device attribute with its directory (optional), e.g. p00/pilatus/1/LastImagePath</string>
       </property>
       <property name="text">
        <string>Dir Attr:</string>
       </property>
       <property name="buddy">
        <cstring>dirLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="dirLabel">
       <property name="toolTip">
        <string>tango device attribute with its filename, e.g. p00/pilatus/1/LastImageTake</string>
       </property>
       <property name="text">
        <string>File Attr:</string>
       </property>
       <property name="buddy">
        <cstring>fileLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="cStatusLabel">
       <property name="toolTip">
        <string>connection status and via which port the zmq security stream is being emitted</string>
       </property>
       <property name="text">
        <string>Status:</string>
       </property>
       <property name="buddy">
        <cstring>cStatusLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="pickleTopicLabel">
       <property name="toolTip">
        <string>ZMQ stream topic</string>
       </property>
       <property name="text">
        <string>DataSource:</string>
       </property>
       <property name="buddy">
        <cstring>pickleTopicComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="pickleLineEdit">
       <property name="toolTip">
        <string>zmq server, port and topic, hwm (optional): server:port[/topic][/hwm]
e.g. haso228:9999/10001 or :5553</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="httpLabel">
       <property name="toolTip">
        <string>monitor url address or hostname/api_version, e.g. http://haso228eiger/monitor/api/1.5.0/images/monitor or haso228eiger/1.5.0</string>
       </property>
       <property name="text">
        <string>URL:</string>
       </property>
       <property name="buddy">
        <cstring>httpLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="sourceTypeComboBox">
       <property name="toolTip">
        <string>image source type, e.g. Hidra, Tango, ZMQ Stream HTTP response, Test</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font: bold;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
